lib/rules/custom-property-no-missing-var-function/index.js

```diff
@@ -0,0 +1,76 @@
+// @ts-nocheck
+
+'use strict';
+
+const valueParser = require('postcss-value-parser');
+
+const declarationValueIndex = require('../../utils/declarationValueIndex');
+const isCustomProperty = require('../../utils/isCustomProperty');
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+
+const ruleName = 'custom-property-no-missing-var-function';
+
+const messages = ruleMessages(ruleName, {
+	rejected: (customProperty) => `Unexpected missing var function for "${customProperty}"`,
+});
+
+function rule(actual) {
+	return (root, result) => {
+		const validOptions = validateOptions(result, ruleName, {
+			actual,
+		});
+
+		if (!validOptions) return;
+
+		const customProperties = new Set();
+
+		root.walkAtRules(/property/i, (atRule) => {
```

Comment:
[question] Does it make sense to check an at-rule name more strictly as below?

```suggestion
		root.walkAtRules(/^property$/i, (atRule) => {
```

Reply:
Good catch. It's a mistake.